v2/brigadier/src/jobs.ts

```diff
@@ -20,33 +61,177 @@ export class Job {
+  }
+
+  /**
+   * Specifies a Runnable that consists of sub-Runnables (such as jobs
+   * or sequential groups) running concurrently.
+   * When run, all Runnables are started simultaneously (subject to
+   * scheduling constraints).
+   * The concurrent group completes when all Runnables have completed.
+   * 
+   * @param runnables The work items to be run in parallel
+   */
+  public static concurrent(...runnables: Runnable[]): ConcurrentGroup {
+    return new ConcurrentGroup(...runnables)
+  }
 }
 
 export class Container {
+  /** The OCI image that the container should run. */
   public image: string
+  /**
+   * Specifies under what conditions the host node should re-fetch an image
+   * that is already in its local cache. The permitted values are as IfNotPresent
+   * (the default) and Always.
+   */
   public imagePullPolicy = "IfNotPresent"
+  /** The working directory for the process running in the container. */
   public workingDirectory = ""
+  /**
+   * The command to run in the container. If not specified, the default entry point
+   * of the container is called.
+   * 
+   * Only the first element of the array is the actual command. Subsequent elements
+   * are treated as arguments. For example, a command of ["echo", "hello"] is
+   * equivalent to running 'echo hello'. A common convention is to use the command
+   * array for subcommands and the arguments array for argument values.
+   * 
+   * @example
+   * job.command = ["helm", "install"]
+   * job.arguments = ["stable/nginx", "-g"]
+   */
   public command: string[] = []
+  /**
+   * The arguments to pass to Container#command.  If the command includes arguments
+   * already, the arguments property are appended. A common convention is to use the command
+   * array for subcommands and the arguments array for argument values.
+   * 
+   * @example
+   * job.command = ["helm", "install"]
+   * job.arguments = ["stable/nginx", "-g"]
+   * 
+   */
   public arguments: string[] = []
+  /** Environment variables to set in the container. */
   public environment: { [key: string]: string } = {}
+  /**
+   * The path in the container's file system where, if applicable, the
+   * Brigade worker's shared workspace should be mounted. If empty (the default),
+   * the Job does not have access to the shared workspace.
+   */
   public workspaceMountPath = ""
+  /**
+   * The path in the container's file system where, if applicable,
+   * source code retrieved from a version control system repository should be
+	 * mounted. If empty (the default), Brigade will not mount any source
+   * code automatically.
+   */
   public sourceMountPath = ""
+  /**
+   * Whether the container should run with privileged permissions. This is
+   * typically required only for "Docker in Docker" scenarios where the
+   * container must run its own Docker daemon.
+   * 
+   * Privileged execution may be disallowed by Brigade project configuration.
+   * If so, the container will run unprivileged.
+   */
```

Comment:
Spot on. Good context too with the "Docker in Docker" explanation.

Reply:
I copied it from your docs for the Go SDK!